packages/core/src/tracing/instrumentation/protocols/http2Server.js

```diff
@@ -31,6 +31,26 @@ exports.init = function init(config) {
   extraHttpHeadersToCapture = config.tracing.http.extraHttpHeadersToCapture;
 };
 
+exports.updateConfig = function updateConfig(config) {
+  extraHttpHeadersToCapture = config.tracing.http.extraHttpHeadersToCapture;
+};
+
+exports.activate = function activate(extraConfig) {
+  if (
+    extraConfig &&
+    extraConfig.tracing &&
+    extraConfig.tracing.http &&
```

Comment:
Suggestion: Sad we cannot use optional chaining. But maybe `utils.get(extraConfig, 'tracing.http')` (see https://lodash.com/docs#get)

Reply:
Yes, optional chaining would be great here but it is only available once we drop Node < 14.

TBH, I find the extra boiler plate okay-ish here. We can revisit the utils.get idea later, for now I would not go back and introduce a util for that here.